services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_pipelines.py

```diff
@@ -25,7 +25,7 @@ async def get_pipeline(self, project_id: ProjectID) -> CompPipelineAtDB:
             )
             row: RowProxy | None = await result.fetchone()
         if not row:
-            raise PipelineNotFoundError(str(project_id))
+            raise PipelineNotFoundError(pipeline_id=project_id)
```

Comment:
TIP: useful context is `user_id` or `project_id`

Reply:
not sure what you mean here. this is a pydantic v2 branch.